chat.lua

```diff
@@ -0,0 +1,29 @@
+
+minetest.register_chatcommand("travelnet", {
+    params = "[network?]",
+    description = "Shows the travelnet formspec for the network",
+    privs = {
+        teleport = true
+    },
+    func = function(playername, network_name)
+        if not network_name or network_name == "" then
```

Comment:
Could add a `:trim()` to prevent whitespace issues

Reply:
i was originally worried about `nil` getting in there but it turns out you always get a string